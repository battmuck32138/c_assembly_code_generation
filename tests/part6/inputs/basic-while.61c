int main () {
  int i = 0;
  while (i <= 10) {
    i = i + 2;
  }
  return i;
}
