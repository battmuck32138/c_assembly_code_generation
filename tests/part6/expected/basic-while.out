
Exited with error code 12
